### ISSUE TITLE
[BUG] docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].

### ISSUE THREAD

**Issue by ithakaa** [JSchmie/ScrAIbe-WebUI#42]
I am attempting to install ScrAIbe-WebUI  inside an Ubuntu 22.04 LXC

`docker run -d --name scraibe-webui -p 7860:7860 --gpus 'all' -v $(pwd)/data:/data hadr0n/scraibe-webui"`

**`docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].`**

I do not have a dedicated GPU so am attempting this with a CPU only setup

**Comment by JSchmie** [comment]
Hi, I think this issue originates in your `docker run` you are applying the `--gpu 'all' ` option, which tells docker to use a GPU. Try to remove the `--gpus 'all'` Option from your docker run command. 
So your command would be: 
```bash
docker run -d --name scraibe-webui -p 7860:7860  -v $(pwd)/data:/data hadr0n/scraibe-webui"
```

If this works, please let us know, then we will try to enhance our docs to avoid this problem in the future.  :)

**Comment by ithakaa** [comment]
Thank you, that did the trick, however...

If I am in the process of transcribing an audio file and I reload the webpage, the task is no longer listed as being processed, what happens to the active task ?

Also, after the task is complete, is there a download transcription option ?

**Comment by JSchmie** [comment]
### Regarding your questions:

> 1. If I am in the process of transcribing an audio file and I reload the webpage, the task is no longer listed as being processed. What happens to the active task?

The active task will continue to be processed in the background, but it will no longer appear in your front end after a page reload. Once the task is finished, the associated file should be deleted from your `tmp` folder. This behavior occurs because Gradio, as mentioned in this [issue](https://github.com/gradio-app/gradio/issues/3106), does not persist session data across page reloads. Additionally, the app doesn't use cookies or similar methods to track sessions, so each page refresh is treated as a new session. However, by using our `asynchronous` backend, you can avoid this issue. In that case, the file and email are linked to the session data, and the processed file will be emailed to you once the task is complete.

> 2. After the task is complete, is there a download transcription option?

Currently, there isn't a specific download button for the transcription. However, since the transcription is just plain text, you can use the copy button above each field to manually save it to a file. Alternatively, using our `asynchronous` backend will provide you with both a `.txt` and a `.json` file when the transcription is complete.

> 3. Transcribing a large 1.5GB audio file results in an "error" on the webpage.

Thank you for reporting this issue. Unfortunately, I haven’t been able to reproduce the error with a large file. Could you please submit another issue using our [bug report template](https://github.com/JSchmie/ScrAIbe-WebUI/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=%5BBUG%5D), providing more details about your system, network configuration, and any relevant Docker logs? This will help us investigate the problem more effectively.

**Comment by ithakaa** [comment]
> However, by using our `asynchronous` backend, you can avoid this issue. In that case, the file and email are linked to the session data, and the processed file will be emailed to you once the task is complete.

What `asynchronous` backend, and email option are you referring too, I don't see email option anywhere.
